### PR TITLE
ppc64le support.

### PR DESCRIPTION
**Pull request by asowani**
Adding support to build Swift on ppc64le platform.

### CODE REVIEW

File: nodes/ppc64le_16_04.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "email": "sowania@us.ibm.com",
+    "company": "IBM"
+  },
+  "node": {
+      "platform": "ppc64le",
+      "os_version": "Ubuntu 16.04"
+  },
+  "jobs": [
+    {
+      "display_name": "Swift - ppc64le (Tools RA, Stdlib RD) swift-4.1-branch)",
```
Comment (shahmishal) [inline nodes/ppc64le_16_04.json]: Can you add `(` for  `swift-4.1-branch)`?

**Comment by asowani** [comment]
@shahmishal Sorry for the typo, I have corrected it now. Thanks!

**Comment by shahmishal** [comment]
Job - https://ci-external.swift.org/job/oss-swift-4.1-RA-linux-ubuntu-16.04-ppc64le/
Node - https://ci-external.swift.org/computer/ubuntu-16.04-ppc64le/

@asowani Thanks!

**Comment by asowani** [comment]
@shahmishal Thanks for adding ppc64le build support!

**Comment by shahmishal** [comment]
@asowani The node has been offline for a while, is this node/job still need it?

**Pull request merged**